libmamba/CMakeLists.txt

```diff
@@ -48,19 +47,15 @@ message(STATUS "libmamba binary version: v${LIBMAMBA_BINARY_VERSION}")
 
 option(BUILD_TESTS "Build libmamba C++ tests" OFF)
 option(BUILD_SHARED "Build shared libmamba library" OFF)
-option(BUILD_STATIC "Build static libmamba library" OFF)
-option(BUILD_STATIC_DEPS "Build static libmamba library with static linkage to its dependencies" OFF)
+option(BUILD_STATIC "Build static libmamba library with static linkage to its dependencies" OFF)
```

Comment:
Should we take this opportunity to introduce an prefix on options: `MAMBA_BUILD_STATIC` *etc.* ?

Reply:
I don't know, `MAMBA_BUILD_MICROMAMBA` sounds a bit weird. And there is already a lot to type to build micromamba for instance

Reply:
`MAMBA_BUILD_MICRO`?

Reply:
still weird. And I don't think we need this prefix as long as this is not used as a framework in another repo building with cmake.